packages/trader/src/sass/add-or-manage.scss

```diff
@@ -88,11 +88,13 @@
 
         & .btn {
             height: 4rem;
-            @include typeface(--title-left-bold-black);
-            font-size: 1.4rem;
-            text-align: center;
-            color: var(--text-colored-background);
             margin: 0 2rem;
+
+            &__text {
+                @include typeface(--title-left-bold-black);
```

Comment:
i think we can just change title to paragraph to make it 14px? or we can use var(--text-size-xs) to make it 1.4rem